### PR TITLE
Link to v6 plan in the README

### PR DESCRIPTION
**Pull request by echarles**
This PR add useful information to the README for the V6 development as discussed during the last community meeting.

cc/ @afshin @jtpio

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,9 @@ computing.
 
 The Jupyter Notebook project is currently undertaking a transition to a more modern code base built from the ground-up using JupyterLab components and extensions.
 
-There is new stream of work which was submitted and then accepted as a Jupyter Enhancement Proposal (JEP): https://jupyter.org/enhancement-proposals/79-notebook-v7/notebook-v7.html
+There is new stream of work which was submitted and then accepted as a Jupyter Enhancement Proposal (JEP) as part of the next version (v7): https://jupyter.org/enhancement-proposals/79-notebook-v7/notebook-v7.html
+
+There is also a plan to move forward the current version (v6): https://github.com/jupyter/notebook-team-compass/issues/5#issuecomment-1085254000
```
Comment (jtpio) [inline README.md:19]: Maybe something like the following to not give the false impression v6 will actively be maintained?

```suggestion
There is also a plan to continue maintaining Notebook v6 with bug and security fixes only, to ease the transition to Notebook v7: https://github.com/jupyter/notebook-team-compass/issues/5#issuecomment-1085254000
```

**Comment by echarles** [comment]
@jtpio Thx for the review. I have committed your suggestions.

**Comment by jtpio** [comment]
Thanks!

**Comment by jtpio** [comment]
CI failure is relevant because the `6.5.x` branch does not exist yet:

image

@echarles let's wait until that branch is created before merging this? Or omit the link for now so the CI check doesn't fail?

**Comment by echarles** [comment]
The link to 6.5.x branch is now removed

**Pull request merged**